src/ucp/core/ucp_request.c

```diff
@@ -34,7 +34,7 @@ ucs_status_t ucp_request_test(void *request, ucp_tag_recv_info_t *info)
     return UCS_INPROGRESS;
 }
 
-void ucp_request_release(void *request)
```

Comment:
should not remove the implementation of ucp_request_release() - need to keep both functions